azure-pipelines-templates/device-test.yml

```diff
@@ -0,0 +1,156 @@
+parameters:
+  - name: appComponents
+    displayName: "Component name"
+    type: object
+  - name: unitTestRunsettings
+    displayName: |
+      Usage: unitTestRunsettings,unitTestDll
```

Comment:
Interesting... I hand't seen this usage of display name to show information about the parameter! :)

Reply:
I've been digging quite a lot in ADO! It's very very very nice!